tcs-persistence/pom.xml

```diff
@@ -18,7 +18,7 @@
     <dependency>
       <groupId>com.transformuk.hee.tis</groupId>
       <artifactId>tcs-api</artifactId>
-      <version>6.16.1</version>
+      <version>6.16.2</version>
```

Comment:
I have a question about the version bump.
The dependency `tcs-api` is updated here, should we bump the version of `tcs-persistence`, and its reference in `tcs-service` to let `tcs-service` know there's a new version?